CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 ### Added
 
 - Added class .action-toggle to .action-list, selecting on .material-icons is deprecated.
+- Added class .panel-header and .panel-footer to panel. Using just <header> and <footer> is still valid.
```

Comment:
Considering the addition off a deprecation warning, maybe it should be more clearly reflected here?